src/components/Dropdown/Dropdown.tsx

```diff
@@ -87,7 +82,11 @@ const ValueItem = styled(Div)`
 `;
 
 const OptionsContainer = styled(Div)`
-  ${({ color, variant, isOpenedBelow, isHidden }: UsefulDropdownState) => `
+  ${({ color, variant, isOpenedBelow, isHidden }: UsefulDropdownState) => {
+    const direction = isOpenedBelow ? 'top' : 'bottom';
+    const borderStyle = isOpenedBelow ? '0rem 0rem 0.25rem 0.25rem' : '0.25rem 0.25rem 0rem 0rem';
```

Comment:
`border` style usually refers to the shorthand border CSS rule, maybe we can call it border radii or something.